### PR TITLE
System.Reflection.Metadata: Remove FUTURE compilation symbol

### PR DESCRIPTION
**Pull request by tmat**

### CODE REVIEW

**Comment by tmat** [comment]
@nguerrera @ericstj

File: src/System.Reflection.Metadata/src/System.Reflection.Metadata.csproj [hunk 1]
```diff
@@ -12,18 +12,8 @@
...
   </PropertyGroup>
-  <ItemGroup>
-    <PackageDestination Include="lib/netstandard1.1">
-      <TargetFramework>netstandard1.1</TargetFramework>
-    </PackageDestination>
-    <!-- Support targets that were supported in previous package versions -->
-    <PackageDestination Include="lib/portable-net45+win8">
-      <TargetFramework>portable-net45+win8</TargetFramework>
-    </PackageDestination>
-  </ItemGroup>
```
Comment (nguerrera) [inline src/System.Reflection.Metadata/src/System.Reflection.Metadata.csproj]: Why is this deleted? I don't know how this works (deferring to @ericstj), but doesn't seem related.

**Comment by nguerrera** [comment]
LGTM apart from the 1 question.

File: src/System.Reflection.Metadata/src/System.Reflection.Metadata.csproj (hunk 1 above)
Comment (tmat) [inline src/System.Reflection.Metadata/src/System.Reflection.Metadata.csproj]: Oh, it shouldn't be. Will fix.

Comment (tmat) [inline src/System.Reflection.Metadata/src/System.Reflection.Metadata.csproj]: I have reverted this change.

**Comment by nguerrera** [comment]
:+1:

**Comment by stephentoub** [comment]
@dotnet-bot test this please

**Comment by ericstj** [comment]
LGTM
